qutebrowser/browser/webengine/webview.py

```diff
@@ -173,7 +173,12 @@ def _set_bg_color(self):
         col = config.val.colors.webpage.bg
         if col is None:
             col = self._theme_color
-        self.setBackgroundColor(col)
+        if self.view() is not None:
+            if col.alpha() < 255:
+                self.view().setAttribute(Qt.WA_TranslucentBackground)
+            else:
+                self.view().resetAttribute(Qt.WA_TranslucentBackground)
+        self.setBackgroundColor(Qt.transparent)
```

Comment:
This now always makes the color transparent, no matter what the user actually configured...

Reply:
Ok I modified it with : 
self.view().setAttribute(Qt.WA_TranslucentBackground, False)